src/reducers/columnStateReducer.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule columnStateReducer
+ */
+
+'use strict';
+
+import ActionTypes from 'ActionTypes'
+import columnStateHelper from 'columnStateHelper'
+
+const DEFAULT_STATE = {
```

Comment:
There's a bunch of stuff here which might not actually be used by the renderers anymore... I should try to clean it up and organize it better.